extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1082,7 +1099,7 @@ static public function substVariables( $cv_value, $city_id ) {
 						/**
 						 * get value for key
 						 */
-						$val = self::getVarValueByName( ltrim( $key, '$' ), $city_id );
+						$val = static::getVarValueByName( ltrim( $key, '$' ), $city_id );
 						if ( $val ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS134)